news/_posts/2025-07-18-gtn-workflowhub.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "GTN is now integrated with WorkflowHub"
+contributions:
+  authorship: [supernord, shiltemann, bgruening]
+tags: [gtn, workflowhub, workflows, fair]
```

Comment:
are there funding tags in the GTN pages? so we can acknowledge ESG & Australian BioCommons